.flowconfig

```diff
@@ -1,7 +1,7 @@
 [ignore]
 
 # We fork some components by platform.
-.*/*.android.js
+.*/*[.]android.js
```

Comment:
FYI `.` as in regex matches any character except a space. Author probably meant a dot.